packages/ui/src/components/Tag/index.tsx

```diff
@@ -106,5 +125,68 @@ export const Tag = ({
       />
     ) : null}
     {isLoading ? <Loader active size={16} /> : null}
-  </StyledContainer>
+  </>
 )
+
+export const Tag = ({
+  children,
+  isLoading,
+  onClose,
+  icon,
+  copiable = false,
+  copyText = 'Copy',
+  copiedText = 'Copied!',
+  disabled,
+  variant = 'neutral',
+  className,
+  'data-testid': dataTestId,
+}: TagProps) => {
+  const [isCopied, setCopied] = useClipboard(
+    typeof children === 'string' ? children : '',
```

Comment:
Should disable copy if it's not string then?

Reply:
It's already done line 151